src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -280,6 +281,14 @@ public async Task ThrowsIfTargetNotSet()
         await Assert.ThrowsAsync<RemoteMethodNotFoundException>(() => this.serverRpc.InvokeAsync(nameof(Server.OverloadedMethod)));
     }
 
+    [Fact]
+    public async Task InvokeWithProgressParameter_NoMemoryLeakConfirm()
+    {
```

Comment:
Please add a `NotifyAsyncWithProgressParameter_NoMemoryLeakConfirm` test as well. 
hint: it will throw an exception since callers aren't allowed to pass Progress objects in Notify methods, but we want to confirm that there is no leak after that exception is thrown.